OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs

```diff
@@ -131,6 +131,21 @@ public IngameChatLogic(Widget widget, OrderManager orderManager, World world, Mo
 
 			CloseChat();
 			chatText.IsDisabled = () => world.IsReplay;
+
+			var keyListener = chatChrome.Get<LogicKeyListenerWidget>("KEY_LISTENER");
+			keyListener.OnKeyPress = e =>
+			{
+				if (e.Event == KeyInputEvent.Up || !chatText.IsDisabled())
+					return false;
+
+				if ((e.Key == Keycode.RETURN || e.Key == Keycode.KP_ENTER) && e.Modifiers == Modifiers.None)
```

Comment:
I just noticed that the regular chat input also binds escape to close.  Please add ` || e.Key == Keycode.ESCAPE` here, and this will be perfect.